src/WebUI/Services/ServiceDetailsView.scss

```diff
@@ -2,13 +2,12 @@
 
 .s-details {
     margin-top: 10px;
-
+    width: auto;
     .s-key {
-        color: $secondary-text;
 
         &.bolt-table-cell {
```

Comment:
&.bolt-table-cell {
            padding-left: 0px;
            line-height: 20px;
        }
    }

    .bolt-list-cell-content {
        padding-left: 0px !important;

        .bolt-list-cell-text {
            padding: 0px !important;
        }
    }

    i do not like these hard coding of the platform, we should see to avoid, informed to @nidhi, but she mentioned we can not remove, we should think about it.

Reply:
I'll explore the possibility of using `className` prop of `ITableColumn`